.flake8

```diff
@@ -0,0 +1,11 @@
+[flake8]
+max-line-length = 100
+max-doc-length = 100
+extend-ignore = E203
+exclude =
+    .git,
+    .venv,
+    __pycache__,
+    old,
+    build,
+    dist
```

Comment:
I suggest using https://pypi.org/project/Flake8-pyproject/ to keep as much configuration as possible in pyproject.toml and don't create too many files

Reply:
Currently resigning from this tool because of it's low maturity. Might switch later.